standalone-metastore/pom.xml

```diff
@@ -365,6 +366,11 @@
         <artifactId>spring-core</artifactId>
         <version>${spring.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.xerial.snappy</groupId>
+        <artifactId>snappy-java</artifactId>
+        <version>${snappy.version}</version>
+      </dependency>
```

Comment:
Why are we defining here? We haven't excluded snappy in this pom either?

Reply:
snappy is coming in only transitively from hadoop-common inside sub-modules of standalone-metastore. 
Changing in the parent pom's dependency management does not effect on this inclusion. So have added in here.